config/application.rb

```diff
@@ -1,33 +1,21 @@
 require_relative 'boot'
 
 require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_view/railtie"
-require "action_cable/engine"
-require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require_relative "boot"
```

Comment:
em là gì mà sửa vào file này thế?

Reply:
Em sửa theo cái sample app vì em thấy hình như require "rails/all" nó thay thế cho được tất cả.

Reply:
Với lại em mới chuyển qua dufng subline nên toàn queen chưa ấn save không như vscode nó toàn tự ấn save nên code chỗ lưu chỗ k lưu thành ra nó lộn xộn ạ

Reply:
để clear ra đi, sau này có gì cần disable thì còn dễ

Reply:
dạ vâng